mlkem/kem.h

```diff
@@ -33,8 +34,105 @@
 #define crypto_kem_enc_derand MLK_NAMESPACE_K(enc_derand)
 #define crypto_kem_enc MLK_NAMESPACE_K(enc)
 #define crypto_kem_dec MLK_NAMESPACE_K(dec)
+
+#define crypto_kem_marshal_pk MLK_NAMESPACE_K(marshal_pk)
+#define crypto_kem_parse_pk MLK_NAMESPACE_K(parse_pk)
+#define crypto_kem_marshal_sk MLK_NAMESPACE_K(marshal_sk)
+#define crypto_kem_parse_sk MLK_NAMESPACE_K(parse_sk)
+#define crypto_kem_keypair_derand_struct MLK_NAMESPACE_K(keypair_derand_struct)
+#define crypto_kem_keypair_struct MLK_NAMESPACE_K(keypair_struct)
+#define crypto_kem_enc_derand_struct MLK_NAMESPACE_K(enc_derand_struct)
+#define crypto_kem_enc_struct MLK_NAMESPACE_K(enc_struct)
+#define crypto_kem_dec_struct MLK_NAMESPACE_K(dec_struct)
```

Comment:
This should be `MLK_ADD_LEVEL` for the structs

Reply:
Not sure I can follow. For the struct I am using using `MLK_ADD_LEVEL`,

Reply:
True